resource.go

```diff
@@ -141,11 +142,11 @@ func ensureDefaultPatches(clusterStatus providerv1alpha1.StatusCluster, patches
 	nodesEmpty := clusterStatus.Nodes == nil
 	versionsEmpty := clusterStatus.Versions == nil
 
-	if conditionsEmpty && nodesEmpty && versionsEmpty {
+	if reflect.DeepEqual(clusterStatus, providerv1alpha1.StatusCluster{}) {
 		patches = append(patches, Patch{
 			Op:    "add",
-			Path:  "/status",
-			Value: Status{},
+			Path:  "/status/cluster",
```

Comment:
For my understanding. So we don't need to add `/status` because it's always already there?